### ISSUE TITLE
insert picture in mail (not as attachment)

### ISSUE THREAD

**Issue by He-Zu** [thunderbird/thunderbird-android#7161]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

6.710

### Problem you are trying to solve

Hello, first of all thank you for the app.

I still have an idea to make the program even better.
the possibility to include a picture directly in the mail (not as an attachment) would be perfect.

many greetings

### Suggested solution

-

### Screenshots / Drawings / Technical details

-

**Comment by EmmanuelDav** [comment]
Hi, I'm interested in working on this issue. Is it still available, if yes can you clarify on the type of picture you think, like a profile picture or ??

**Comment by He-Zu** [comment]
Hello, I would like to embed a screenshot or jpg/png file directly into the mail. Not as an attachment. Thunderbird on the desktop computer can do this with cut and paste. see attached picture

![2023-10-05_16-12](https://github.com/thundernest/k-9/assets/116963345/a6ce3d4e-d410-42e3-a258-495fc8714da9)

**Comment by nylimited** [comment]
This 👆 is the feature still holding me back from using this app as a daily driver. I am still using Aquamail which has had this feature for a very long time.

**Comment by He-Zu** [comment]
@nylimited I also used Aquamail for a long time. In principle a very good mail program - BUT - have you ever checked how many trackers are built into Aquamail. All the Google Analytics stuff and a lot more data collection that I don't want.
That's not an option for me, which is why I use K9.
Maybe the feature with enbeded Picture will be built in.
Best regards

**Comment by nylimited** [comment]
@He-Zu 

Yes, indeed. Long ago Aquamail was fairly clean. Ironically I went to Aquamail from an old K-9 mail because I needed in-line images. These days Aquamail is seldom used by me. I do have K-9 installed along with Aquamail. My primary email program is Thunderbird on Linux. I am definitely rooting for K-9 or TB mobile that will let me switch completely to the TB family. 

I think we are on the same page. 

Cheers!

**Comment by kewisch** [comment]
Covering this in #7373

**Comment by mxboarder** [comment]
hello
like the original post said, thank you for the app!

but i do not agree that this is a duplicate of 7373.

7373 is about the camera

the original post & my same issue is: 

- *insert* an image "inline" in an email, rather than as an attachment

example: send a screenshot inline in an email to explain it directly

that "feature" is still not available, and it would be great if it were added

thank you!
